scripts/build/rollup.ts

```diff
@@ -1,35 +1,20 @@
-import {BundleDef, BundleType} from "./bundles"
-import {asyncCopyTo, getPackageDir} from "./utils"
-import * as Modules from "./modules"
-import * as Packaging from "./packaging"
-import * as Stats from "./stats"
-
-import * as fs from "fs";
-import * as path from "path";
-import sizes from "./plugins/sizes";
-
-const { rollup } = require("rollup")
-const babel = require("rollup-plugin-babel")
-const { terser } = require("rollup-plugin-terser")
-const commonjs = require("rollup-plugin-commonjs")
-const resolve = require("rollup-plugin-node-resolve")
-const prettier = require("rollup-plugin-prettier")
-const replace = require("rollup-plugin-replace")
-const typescript = require("rollup-plugin-typescript2")
-const ora = require("ora")
-const chalk = require("chalk")
-const argv = require("minimist")(process.argv.slice(2))
-const codeFrame = require("babel-code-frame")
-const babelConfig = require("./babel.config")
-
-enum RollupBundleType {
-  UMD_DEV= "UMD_DEV",
-  UMD_PROD= "UMD_PROD",
-  NODE_DEV= "NODE_DEV",
-  NODE_PROD= "NODE_PROD",
-  ESM_PROD= "ESM_PROD",
-  ESM_DEV= "ESM_DEV",
-}
+import {BundleDef} from "./bundles"
+import {getPackageDir} from "./utils"
+
+import * as fs from "fs"
+import * as path from "path"
+import dts from 'rollup-plugin-dts'
+
+import {ModuleFormat, rollup, RollupBuild, RollupError, RollupOptions, RollupWarning} from "rollup"
+import babel from "@rollup/plugin-babel"
+import commonjs from "@rollup/plugin-commonjs"
+import resolve from "@rollup/plugin-node-resolve"
+import babelConfig from './babel.config';
+
+import ora from "ora";
+import chalk from "chalk";
```

Comment:
also i tried to update `chalk` to the latest version, but unfortunately it distributed as ESM package and you can not import ESM package from cjs code. So we stick with this version for a while.